### PR TITLE
Fix panics

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>

**What this PR does / why we need it**:
This PR removes the panic calls from capdctl

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #86 

**Special notes for your reviewer**:
This also deletes some unused code simplifying

/assign @ashish-amarnath 

```release-note
NONE
```

### CODE REVIEW

File: cmd/capdctl/main.go [hunk 1]
```diff
@@ -99,23 +97,67 @@ func main() {
 
 	switch os.Args[1] {
 	case "setup":
-		setup.Parse(os.Args[2:])
-		makeManagementCluster(*managementClusterName, *version, *capdImage, *capiImage)
+		if err := setup.Parse(os.Args[2:]); err != nil {
+			fmt.Printf("%+v\n", err)
+			os.Exit(1)
```
Comment (vincepri) [inline cmd/capdctl/main.go]: klog.Fatal?

Comment (chuckha) [reply cmd/capdctl/main.go]: i'm not actually sure why klog is in this file. If we need a logger we will use klogr. I'm going to play around with ripping out klog here and see why it exists in the first place.

**Comment by chuckha** [comment]
@vincepri ah ha, i found a few more panics and set up a klogr to pass through the code and use it as a setup logger.

**Comment by chuckha** [comment]
yeah lots of repetition right now, if it becomes a problem we can solve it later

**Pull request merged**